app/controllers/request-psp-test-account/post.controller.js

```diff
@@ -35,15 +26,30 @@ module.exports = async function submitRequestForPspTestAccount (req, res, next)
     const pageData = {}
 
     if (service.currentPspTestAccountStage === NOT_STARTED || !service.currentPspTestAccountStage) {
-      await submitRequestAndUpdatePspTestAccountStatus(req)
-      pageData.pspTestAccountRequestSubmitted = true
+      const gatewayAccount = await connectorClient.getAccountByServiceIdAndAccountType({ serviceId: service.externalId, accountType: 'test' })
+      if (gatewayAccount.payment_provider === 'sandbox') {
+        const sandboxGatewayAccountId = gatewayAccount.gateway_account_id
+        const gatewayAccountExternalId = await submitRequestAndUpdatePspTestAccountStatus(req)
+
+        try {
+          await publicAuthClient.revokeTokensForAccount(sandboxGatewayAccountId)
+        } catch (error) {
+          logger.error(`There was an error revoking tokens for sandbox account with id ${sandboxGatewayAccountId}. ${error}`)
```

Comment:
I'm of the opinion that a failed call to revoke tokens for account shouldn't stop the page flow for requesting a new Stripe test account, which is why I've simply caught an error here.